contracts/Market.sol

```diff
@@ -0,0 +1,94 @@
+pragma solidity ^0.4.24;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/lifecycle/Pausable.sol";
+
+import "./IMarket.sol";
+import "./Property.sol";
+
+// @title Radical Bodies market.
+contract Market is IMarket, Ownable, Pausable {
+    Property private _token;
```

Comment:
Probably better to use IERC721 here

Reply:
Or make a custom interface

Reply:
Was thinking about that but it does need the enumerable + metadata interface.